src/Resources/app/storefront/src/js/filter-category-select.plugin.js

```diff
@@ -84,7 +105,7 @@ export default class FilterCategorySelectPlugin extends FilterBasePlugin {
         Object.keys(params).forEach(key => {
             if (key === this.options.name) {
                 stateChanged = true;
-                const ids = params[key].split('_');
```

Comment:
Keep the ending statement semi-colon for consistency with the existing code.

Reply:
@D3xter7 This does not seem to be changed. I see some other lines as well without the ending semi-colon.